CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules")
 ################ PROJECT VERSION ####################
 set(PROJECT_VERSION_FULL "0.1.9")
 set(PROJECT_SO_VERSION 7)
-set(HEX_VERSION_OVERRIDE "0x108") # For CMake < 3.13
+set(HEX_VERSION_OVERRIDE "0x109") # For CMake < 3.13
```

Comment:
I keep thinking I should change that to read `0x000109`, so it's clearer how the "encoding" for the version number is actually done. I know I'm _eventually_ going to screw up trying to increment it, the way it's formatted now.